### ISSUE TITLE
Angular1 | Splitter issue: Weird bug if linebreak used between ons-splitter-content tags

### ISSUE THREAD

**Issue by Chozzle** [OnsenUI/OnsenUI#1912]
__Environment__

  
```
[Core]
  onsenui latest
  
[Framework]
  angular1 1.5.5

[Framework binding]
 angular1-onsenui latest

[Platform]
  Desktop - OS X 10.12.3
  Android device with cordova

[Browser]
  Desktop - Safari 10.0.3 on OS X 10.12.3
  Chrome
```

__Encountered problem__
  A little return keystroke breaks my app entirely and takes me 2 days to fix :(
 The splitter won't open if you click the menu button. It won't close after clicking a menu item. Initial page doesn't load.

__How to reproduce__
Press enter between tags so it looks like: 
<ons-splitter-content>
</ons-splitter-content>
not like:
<ons-splitter-content></ons-splitter-content>

  
  [__Demo link__](https://tutorial.onsen.io/?issue)

  

- __HTML__

```html
<ons-splitter var="mySplitter" ng-controller="SplitterController as splitter">
  <ons-splitter-side side="left" width="220px" collapse swipeable>
    <ons-page>
      <ons-list>
        <ons-list-item ng-click="splitter.load('home.html')" tappable>
          Home
        </ons-list-item>
        <ons-list-item ng-click="splitter.load('settings.html')" tappable>
          Settings
        </ons-list-item>
        <ons-list-item ng-click="splitter.load('about.html')" tappable>
          About
        </ons-list-item>
      </ons-list>
    </ons-page>
  </ons-splitter-side>
  <ons-splitter-content page="home.html">
    
  </ons-splitter-content>
</ons-splitter>

<ons-template id="home.html">
  <ons-page>
    <ons-toolbar>
      <div class="left">
        <ons-toolbar-button ng-click="mySplitter.left.open()">
          <ons-icon icon="md-menu"></ons-icon>
        </ons-toolbar-button>
      </div>
      <div class="center">
        Main
      </div>
    </ons-toolbar>
    <p style="text-align: center; opacity: 0.6; padding-top: 20px;">
      Swipe right to open the menu!
    </p>
  </ons-page>
</ons-template>

<ons-template id="settings.html">
  <ons-page>
    <ons-toolbar>
      <div class="left">
        <ons-toolbar-button ng-click="mySplitter.left.open()">
          <ons-icon icon="md-menu"></ons-icon>
        </ons-toolbar-button>
      </div>
      <div class="center">
        Settings
      </div>
    </ons-toolbar>
  </ons-page>
</ons-template>

<ons-template id="about.html">
  <ons-page>
    <ons-toolbar>
      <div class="left">
        <ons-toolbar-button ng-click="mySplitter.left.open()">
          <ons-icon icon="md-menu"></ons-icon>
        </ons-toolbar-button>
      </div>
      <div class="center">
        About
      </div>
    </ons-toolbar>
  </ons-page>
</ons-template>
```

- __JS__

```javascript
ons.bootstrap()
  .controller('SplitterController', function() {
    this.load = function(page) {
      mySplitter.content.load(page)
        .then(function() {
          mySplitter.left.close();
        });
    };
  });
```

**Comment by frandiox** [comment]
@Chozzle Oops, looks like a wild bug. We'll fix it, thanks for reporting 😅

**Comment by misterjunio** [comment]
@Chozzle we are taking care of this issue but it's trickier than expected and we need to perform some modifications in the core to make it work. Meanwhile you can just keep no space in the `splitter-content`. Amazing the damage a small blank space can do 😄

**Comment by Chozzle** [comment]
I literally compared my app bit by bit to the example code to find it. I didn't think an enter between html tags was "a thing".

**Comment by asial-matagawa** [comment]
@Chozzle 
Browsers treat spaces, line breaks and texts as a text node equally, so it's impossible to give line breaks special treatment.

We are now working on fixing this problem on #1926.

**Comment by frandiox** [comment]
This was fixed a while ago in 2.3.0 :+1:
